src/javascript/app/pages/resources/trading_times/trading_times.ui.js

```diff
@@ -48,23 +48,37 @@ const TradingTimesUI = (() => {
         });
         $date.val(localize('Today'));
         if ($(window).width() < 480) {
-            // Create a label to be friendlier
-            const $label = $('label[for=trading-date]');
-            $label.append($('<span/>', { class: 'ux-date foot-note' }));
+            const $input_group = $('#trading-date-container .input-group');
+
+            const $date_input = $input_group.find('input');
+
+            $input_group.append($('<span/>', { class: 'ux-date foot-note' }));
+
+            const $date_caption = $('span.ux-date');
```

Comment:
Could you move this declaration to other constants above? 🙏🏻

Reply:
Hi @akmal-binary  these variables are created on load of the window so If I declare the variable before the element is created then it will be empty

Reply:
Ah ok, sorry I haven't noticed that it refers to the element above 👍🏻

Reply:
No worries :pray: